src/validator.js

```diff
@@ -245,6 +248,12 @@ export default class Validator {
           return this.scanFiles(cssFiles);
         })
         .then(() => {
+          return this.xpi.getFilesByExt('.html');
```

Comment:
It occurs to me that using multiple extensions, at least for HTML, would make sense.

I could extend the `getFilesByExt` to take multiple extensions or I could make the call several times to each file type. I feel like the latter is cleaner, but it probably adds more code. Any particular opinion? (I'd go for the first option, eg:

```js
this.xpi.getFilesByExt('.html', '.htm');
```